src/queries/schema/GetAllOpenPRsAndCardIDs.ts

```diff
@@ -22,6 +22,10 @@ export interface GetAllOpenPRsAndCardIDs_repository_pullRequests_nodes_projectCa
 
 export interface GetAllOpenPRsAndCardIDs_repository_pullRequests_nodes {
   __typename: "PullRequest";
+  /**
+   * Identifies the pull request branch.
+   */
+  branch: string;
```

Comment:
I couldn't get the schema generation working locally. This is my best inference for what it's supposed to be.

```plaintext
$ npm run graphql-schema

> dt-mergebot@3.0.0 graphql-schema
> npx apollo client:codegen schema --target typescript --globalTypesFile=src/queries/schema/graphql-global-types.ts

    Error: Cannot find module 'graphql/validation/rules/KnownArgumentNamesRule'
    Require stack:
    - /Users/josh/repos/dt-mergebot/node_modules/@apollo/federation/dist/composition/validate/preNormalization/tagDirective.js
    - /Users/josh/repos/dt-mergebot/node_modules/@apollo/federation/dist/composition/validate/preNormalization/index.js
    - ...
    Code: MODULE_NOT_FOUND
```